distribution/docs/src/main/resources/_contents/_metrics/metrics-reporting-contents.adoc

```diff
@@ -0,0 +1,108 @@
+
+==== Metrics Reporting Application
+
+The ${branding} Metrics Reporting application provides access to historical data in a graphic, a comma-separated values file, a spreadsheet, a PowerPoint file, XML, and JSON formats for system metrics collected while ${branding} is running.
+Aggregate reports (weekly, monthly, and yearly) are also provided where all collected metrics are included in the report.
+Aggregate reports are available in Excel and PowerPoint formats.
+
+The ${branding} Metrics Reporting application provides a plugin that adds a new tab to the Admin Console with the title of Metrics.
+When selected, the Metrics tab displays a list of all of the metrics being collected by ${branding}, e.g., Catalog Queries, Catalog Queries Federated, Catalog Ingest Created, etc.
+
+With each metric in the list, a set of hyperlinks is displayed under each column.
+Each column's header is displayed with the available time ranges. The time ranges currently supported are 15 minutes, 1 hour, 1 day, 1 week, 1 month, 3 months, 6 months, and 1 year, measured from the time that the hyperlink is clicked.
+
+All metrics reports are generated by accessing the collected metric data stored in the `<${branding}_INSTALL_DIR>/data/metrics` directory.
+All files in this directory are generated by the JmxCollector using RRD4J, a Round Robin Database for a Java open source product.
+All files in this directory will have the `.rrd` file extension and are binary files, hence they cannot be opened directly.
+These files should only be accessed using the Metrics tab's hyperlinks.
+There is one RRD file per metric being collected.
+Each RRD file is sized at creation time and will never increase in size as data is collected.
+One year's worth of metric data requires approximately 1 MB file storage.
+
+[WARNING]
+====
+Do not remove the `<${branding}_INSTALL_DIR>/data/metrics` directory or any files in the directory.
+If this is done, all existing metrics data will be permanently lost.
+
+Also note that if ${branding} is uninstalled/re-installed, all existing metrics data will be permanently lost.
+====
+
+There is a hyperlink per format in which the metric's historical data can be displayed.
```

Comment:
Revised to: `Hyperlinks are provided for each metric and each format in which data can be displayed.`